CHANGELOG.md

```diff
@@ -14,6 +14,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 - httpfuzzerprocessor/unexpected_responses.js > compare response codes to a (pass/fail) regex and generate alerts
 - targeted/dns-email-spoofing > Check if DMARC / SPF policies are configured on a domain.
 - httpsender/add-more-headers.js > Add caller-specified headers to all requests.
+- authentication/OfflineTokenRefresh.js - refresh oauth2 offline tokens
+- httpsender/AddBearerTokenHeader.js - refresh oauth2 offline tokens
```

Comment:
These should be moved into the Unreleased section:

```md
## [Unreleased]
### Added
- authentication/OfflineTokenRefresh.js - refresh oauth2 offline tokens
- httpsender/AddBearerTokenHeader.js - refresh oauth2 offline tokens
- ```

Reply:
changed, I probably merged it in the wrong place without noticing it

Reply:
You got it in the right spot but missed the `### Added`